libs/ojsonnet/Eval_jsonnet_subst.ml

```diff
@@ -1,13 +1,24 @@
 open Core_jsonnet
 open Common
-module V = Value_jsonnet_subst
+module V = Value_jsonnet
 module A = AST_jsonnet
 module J = JSON
 
 exception Error of string * Tok.t
 
 type cmp = Inf | Eq | Sup
```

Comment:
would be good to  factorize code as much as possible with Eval_jsonnet.ml, for example here reuse the cmp type
from the other file.